Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -187,14 +223,18 @@ YUI.add('ez-roleserversideviewservice', function (Y) {
          * @param {Integer} countAssigned number of successfully assignments
          * @param {Function} callback the callback to call when other tasks are done
          */
-        _assignRoleCallback: function (roleId, roleName, contents, countAssigned, callback) {
+        _assignRoleCallback: function (data, contents, countAssigned, callback) {
```

Comment:
if you change the parameter, you need to change the JSDoc block. Also `data` is a very generic name, you should try to find something a bit more precise.